src/quantum/README.rst

```diff
@@ -0,0 +1,188 @@
+
+   .. code-block::
+
+      az quantum workspace show -o table
+
+      Location     Name                               ResourceGroup
+      -----------  ---------------------------------  --------------------------------
+      westus       ws-yyyyyy                          rg-yyyyyyyyy
+
+
+6. You can see all the jobs submitted to a workspace using `az quantum job list`.
+
+   .. code-block::
+
+      az quantum job list -o table
+
+      Id                                    State    Target          Submission time
+      ------------------------------------  -------  --------------  ---------------------------------
+      yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+
+   The console will output the information about the job, including the ID of the job.
+
+
+7. You can use the ID of the job to track its status.
+
+   .. code-block::
+
+      az quantum job show -id yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy -o table
+
+      Id                                    State    Target          Submission time
+      ------------------------------------  -------  --------------  ---------------------------------
+      yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+
+
+8. Once the job finishes you can visualize the job's results with `az quantum job output`:
+
+   .. code-block::
+
+      az quantum job output -id yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy -o table
+   
+      Result    Frequency
+      --------  -----------  -------------------------
+      [0,0]     0.25000000   ▐█████                  |
+      [1,0]     0.25000000   ▐█████                  |
+      [0,1]     0.25000000   ▐█████                  |
+      [1,1]     0.25000000   ▐█████                  |
+
+
+Submitting Q# programs for execution from the command line
+==========================================================
+
+Prerequisites
+-------------
+
+- You need to have an Azure Quantum workspace in your subscription.
+- Install the [Quantum Development Kit](https://docs.microsoft.com/quantum/install-guide/standalone), if you haven't already.
+
+
+Write your quantum application
+------------------------------
+
+First you need to have the Q# quantum application that you want to execute in
+Azure Quantum.
+
+.. tip::
+   If this is the first time for you to create Q# quantum applications, you can learn how
+   in our [Microsoft Learn module](https://docs.microsoft.com/en-us/learn/modules/qsharp-create-first-quantum-development-kit/).
+
+In this case we will use a simple quantum random bit generator. We create a Q#
+project and substitute the content of `Program.qs` with the following code:
+
+.. code-block::
+
+   namespace RandomBit {
+
+       open Microsoft.Quantum.Canon;
+       open Microsoft.Quantum.Intrinsic;
+       open Microsoft.Quantum.Measurement;
+
+       @EntryPoint()
+       operation GenerateRandomBit() : Result {
+           using (q = Qubit()) {
```

Comment:
The "using" keyword has been replaced with "use", and qubits may now be allocated without a block. Consider "use q = Qubit();" or "use q = Qubit() { ... }".

Reply:
Deprecated syntax. Parentheses here are no longer required and will not be supported in the future.

Reply:
Ah, good catch. Updating the sample here.